### ISSUE TITLE
aws.app-elb and aws.elb missing auto-tag-user action

### ISSUE THREAD

**Issue by tjstansell** [cloud-custodian/cloud-custodian#5379]
**Describe the bug**
The aws.app-elb and aws.elb resources no longer show `auto-tag-user` as an available action.  This seems like it might be a result of the refactor related to #5032.

**To Reproduce**
1. install latest git version of custodian
2. `custodian schema aws.app-elb.actions`

**Expected behavior**
`auto-tag-user` should be an available action.

**Background (please complete the following information):**
 - Python Version: 3.7.6
 - Custodian Version: 0.8.46.1.dev37+gf4980b758
 - Cloud Provider: aws
 - Policy: [please exclude any account/sensitive information]
```yaml
policies:
  - name: app-elb-creator
    resource: app-elb
    actions:
      - type: auto-tag-user
        tag: Creator
```
 - Traceback: [if applicable, please exclude sensitive/account information]
```
$ custodian validate a.yaml
2020-02-26 11:23:16,591: custodian.commands:ERROR Configuration invalid: a.yaml
2020-02-26 11:23:16,591: custodian.commands:ERROR Error on policy:app-elb-creator resource:app-elb
{'type': 'auto-tag-user', 'tag': 'Creator'} is not valid under any of the given schemas

Failed validating 'anyOf' in schema[16]['allOf'][1]['properties']['actions']['items']:
    {'anyOf': [{'$ref': '#/definitions/resources/aws.app-elb/actions/delete'},
               {'$ref': '#/definitions/actions/aws.invoke-lambda'},
               {'$ref': '#/definitions/actions/aws.invoke-sfn'},
               {'$ref': '#/definitions/actions/aws.mark-for-op'},
               {'$ref': '#/definitions/resources/aws.app-elb/actions/modify-listener'},
               {'$ref': '#/definitions/actions/aws.modify-security-groups'},
               {'$ref': '#/definitions/actions/aws.notify'},
               {'$ref': '#/definitions/actions/aws.post-finding'},
               {'$ref': '#/definitions/actions/aws.post-item'},
               {'$ref': '#/definitions/actions/aws.put-metric'},
               {'$ref': '#/definitions/actions/aws.remove-tag'},
               {'$ref': '#/definitions/resources/aws.app-elb/actions/set-s3-logging'},
               {'$ref': '#/definitions/resources/aws.app-elb/actions/set-shield'},
               {'$ref': '#/definitions/resources/aws.app-elb/actions/set-waf'},
               {'$ref': '#/definitions/actions/aws.tag'},
               {'$ref': '#/definitions/actions/aws.webhook'},
               {'enum': ['notify',
                         'webhook',
                         'invoke-lambda',
                         'put-metric',
                         'post-finding',
                         'invoke-sfn',
                         'post-item',
                         'set-shield',
                         'set-waf',
                         'set-s3-logging',
                         'mark-for-op',
                         'tag',
                         'remove-tag',
                         'delete',
                         'modify-listener',
                         'modify-security-groups']}]}

On instance['actions'][0]:
    {'tag': 'Creator', 'type': 'auto-tag-user'}
2020-02-26 11:23:16,592: custodian.commands:ERROR app-elb-creator
```
**Additional context**

**Comment by tjstansell** [comment]
Actually, looking more closely, this is missing from other resource types too ... `asg`, `emr`, `redshift`, `s3` are others that are known to be missing it.

**Comment by kapilt** [comment]
hmm.. thanks, looking now, its possibly an import ordering issue for the provider, we need to explicitly import generic stuff (actions/filters) in the resource loader for it to register with the lazy loaded resources.

**Comment by kapilt** [comment]
Didn’t see cause in a 10m debug session, going to look again tomorrow

**Comment by kapilt** [comment]
okay, I found the issue. we used to call resources type subscribers at different times when loading resources including at the end of resource loading, the new registry subscriber semantic does away with subscribing at different lifecycle times, because with lazy loading there notionally isn't all resources done loading signal. instead subscribers are re-entrant and called as resources are loaded lazily. specifically for auto-tag-user we need the action resource type subscriber to be called after the tag actions are registered. I'll update the resource loading for each resource to re-invoke subscribers
